sequana/fasta.py

```diff
@@ -81,7 +81,7 @@ def next(self):  # python 2
     def __len__(self):
         if self._N is None:
             logger.info("Reading input fasta file...please wait")
-            self._N = len([x for x in FastxFile(self.filename)])
+            self._N = len([1 for x in FastxFile(self.filename)])
```

Comment:
J'aurais plutôt écrit ça comme ça pour éviter la création d'une liste:
`self._N = sum(1 for x in FastxFile(self.filename))`